policy/diamond/policy/token/token.rego

```diff
@@ -2,17 +2,25 @@ package diamond.policy.token
 
 import rego.v1
 
+issuer := opa.runtime().env.ISSUER
+
+jwks_endpoint := jwks_endpoint if {
+	metadata := http.send({
+		"url": concat("", [issuer, "/.well-known/openid--configuration"]),
```

Comment:
We should pass in the well known URL and retrieve the `issuer` from there, as the path of the well-known endpoint isn't guaranteed to be `/.well-known/openid-configuration`

Reply:
Oh, fair enough then. Swear I've seen it be something else but can't figure out where